repos/spack_repo/builtin/packages/parafeed/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+from spack.build_systems.cmake import CMakePackage
```

Comment:
The location of build systems has changed:

```suggestion
from spack_repo.builtin.build_systems.cmake import CMakePackage

from spack.package import *
```

Reply:
With this fix my tool can confirm the homepage and all 4 sha256s.